src/HomePage/components/TdeeCalculator/components/ActivityLevel.tsx

```diff
@@ -0,0 +1,48 @@
+import { IActivityObject } from "../../data/models/ActivityObjectModel";
+
+interface Props {
+  stepTitle: string;
+  object?: IActivityObject[];
+  array?: string[];
+  handleClick?: (
+    level: "sedentary" | "active" | "very_active" | "extra_active"
```

Comment:
can't you use "keyof" or an enum or something for every place where you have these as options? See IActivityObject for context

Reply:
Fixed